js/bundle.go

```diff
@@ -39,6 +40,23 @@ type Bundle struct {
 	moduleResolver  *moduleResolver
 }
 
+// TODO: this is to be removed once this is not a warning and it can be moved to the registry
```

Comment:
Can we open the issue and the milestone and link it here, please? So we will have the proper removal as planned, otherwise, we will forget about it.

Reply:
The issue is https://github.com/grafana/k6/issues/3065 - I did create teh milestone though. 

I don't really see the point in linking it though

Reply:
>I don't really see the point in linking it though

The point is if someone hits the TODO and they do not remember about the issue (like me in this case) or is not aware of it at all, it can quickly get the context if they need it. Just better UX for us (the devs).